tango/integrations/torch/train.py

```diff
@@ -203,6 +203,12 @@ def run(  # type: ignore[override]
             if amp:
                 raise ConfigurationError("AMP requires training on CUDA devices")
 
+        if validate_every is not None and validation_split is None:
```

Comment:
I think you need to change the type of `validate_every` to `Optional[int]` and default to `None`.